### PR TITLE
add company details in verejne sidebar

### PR DESCRIPTION
**Pull request by mirecmrozek**

### CODE REVIEW

**Comment by mirecmrozek** [comment]
Now an Icon next to the name of an item in Verejne sidebar opens modal and searches for exact name.
I left toggling data details from /getInfos, because sometimes getInfos actually provides more data than getInfo.

**Comment by mirecmrozek** [comment]
Please merge once approved.

File: client/src/components/Verejne/Map/AddressDetail/ListRow.js [hunk 1]
```diff
@@ -23,24 +57,43 @@ const DetailedInfo = compose(
...
   ),
-  withDataProviders(({id}) => [entityDetailProvider(id)])
+  withDataProviders(({id}) => [entityDetailProvider(id)]),
+  withHandlers({
+    openModalSearch: ({toggleModalOpen, setEntitySearchFor, data}) => (e) =>{
+      if (e.stopPropagation) e.stopPropagation()
+      setEntitySearchFor(data.name)
+      toggleModalOpen()
+    }
+  })
```
Comment (mpinter) [inline client/src/components/Verejne/Map/AddressDetail/ListRow.js]: now that this is a fully fledged connected component, we should move it to a separate file (we're keeping one connected component per file, only components like 'Item' which are essentially just helpers can be kept alongside others)

File: client/src/components/Verejne/Map/AddressDetail/ListRow.js [hunk 2]
```diff
@@ -23,24 +57,43 @@ const DetailedInfo = compose(
...
       data: entityDetailSelector(state, id),
     }),
-    {toggleEntityInfo}
+    {toggleEntityInfo, toggleModalOpen, setEntitySearchFor},
   ),
-  withDataProviders(({id}) => [entityDetailProvider(id)])
+  withDataProviders(({id}) => [entityDetailProvider(id)]),
+  withHandlers({
+    openModalSearch: ({toggleModalOpen, setEntitySearchFor, data}) => (e) =>{
+      if (e.stopPropagation) e.stopPropagation()
```
Comment (mpinter) [inline client/src/components/Verejne/Map/AddressDetail/ListRow.js]: why this ? does anyone set e.stopPropagation elsewhere ?

Comment (jakubukrop) [reply client/src/components/Verejne/Map/AddressDetail/ListRow.js]: Yes, in EntitySearch - but that is a case of form submit, it should not be needed here.

**Pull request closed without merging**